### ISSUE TITLE
Collection filter display

### ISSUE THREAD

**Issue by ghost** [spokanelibrary/ent#51]
Many of the collection codes are too long to display fully. Is there a way to expand the column, make the text smaller, or maybe shorten the collection name without being too obscure?
![search results for cat - mozilla firefox 1132015 101044 am bmp](https://cloud.githubusercontent.com/assets/15221465/10917206/68313702-8214-11e5-9a8c-372ae4c46ecc.jpg)

**Comment by seangirard** [comment]
Yes, I believe we can setup translations for Horizon collection code names.
